cli/config/configfile/file.go

```diff
@@ -300,7 +300,8 @@ func (configFile *ConfigFile) GetAllCredentials() (map[string]types.AuthConfig,
 	for registryHostname := range configFile.CredentialHelpers {
 		newAuth, err := configFile.GetAuthConfig(registryHostname)
 		if err != nil {
```

Comment:
Do we know what errors this can return? Can we exclude "not found" errors here? (which may be "expected")?

Reply:
We can definitely treat `CredentialsNotFound` error specially, but other errors can occur, and I wonder if we shouldn't skip them regardless.

Reply:
Yes, was thinking about the logs mostly (as it would produce some logs that are mostly expected)

Reply:
Sounds good, I'll change that.